unlock-app/src/hooks/useSIWE.tsx

```diff
@@ -152,40 +163,36 @@ export const SIWEProvider = ({ children }: Props) => {
     }
   }
 
-  const signIn = async () => {
+  const signIn = useCallback(async () => {
```

Comment:
hum, that function does not do anything anymore :D Let's clean it up!

Reply:
okay got it!